aux/gv_results.slim

```diff
@@ -33,16 +33,12 @@ html
             h4.modal-title Incompatible Browser!
           #browseralertbody.modal-body
             p#browseralertText
-              strong#mainbrowseralertText Sorry, this feature is not supported in your browser.
+              strong#mainbrowseralertText Sorry, this feature is not supported when opening the results directly in a browser.
```

Comment:
If it's just the graphs that are disable, maybe better to be specific: 'Sorry, viewing graphs is not supported ...'